src/explorer/editors/FileEditor.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as util from '../../util';
+import { BaseEditor, IAzureNode } from 'vscode-azureextensionui';
+import { FileTreeItem } from '../FileTreeItem';
+import { KuduClient } from '../../KuduClient';
+import { getOutputChannel } from '../../util';
+import { nodeUtils } from '../../utils/nodeUtils';
+
+

```

Comment:
Do you still have "tslint.autoFixOnSave" on? I feel like you've got several extra whitespace lines in your PR, and that should've automatically fixed it for you

Reply:
It's on, but the config setting isn't recognized.  I'll figure this out later.